_pages/faqs.html

```diff
@@ -13,27 +13,40 @@ <h1>{{ page.heading }}</h1>
 </header>
 
 <div class="content">
+  {% comment %}
   <nav class="row column v-pad-bottom faq-nav">
     <h2 class="text-center">The answer to every question below is <strong>YES!</strong></h2>
     <p>Inspired by a <a href="https://m.signalvnoise.com/how-a-rest-stop-on-the-side-of-the-road-inspired-a-web-page-design-c2a65bf2e88a">sign at a roadside rest stop</a>. If you have a question that is not answered below, please <a href="mailto:{{site.contact_us_email}}">let us know</a>!</p>
   </nav>
+  {% endcomment %}
 
   <div class="section-pad theme-medium-gray">
     <div class="row column faq-list">
       <section class="faq-item">
-        <h2 class="text-center">Tickets</h2>
-        <p><em>Tickets are no longer on sale. More information is on our <a href="{% link _pages/tickets.html %}">Tickets</a> page. {% comment %}Conference tickets are sold out for 2022, but tutorial tickets may still be available.{% endcomment %}</em></p>
+        <h2 class="text-center">Conference</h2>
+        <dl>
+          {% comment %}
+          <dt>How large is DjangoCon Europe 2023?</dt>
```

Comment:
For screen reader users, all of those would be better as `h3` elements with paragraphs under, so they can navigate the page question by question.